docs/api/swagger.json

```diff
@@ -69,47 +75,927 @@
         }
       }
     },
-    "/auth": {
+    "/user": {
+      "delete": {
+        "summary": "Delete user",
+        "tags": [
+          "user"
+        ],
+        "operationId": "deleteUser",
+        "security": [],
+        "description": "Nothing needed. UserId gets parsed from JWT",
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/user/password/reset": {
+      "post": {
+        "summary": "Reset user password",
+        "tags": [
+          "auth"
+        ],
+        "operationId": "resetPassword",
+        "security": [],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/ResetPassword"
+        },
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/languagePackage": {
+      "post": {
+        "summary": "Create language package for user",
+        "tags": [
+          "language package"
+        ],
+        "operationId": "createLanguagePackage",
+        "security": [],
```

Comment:
the `security: []` means no auth. But for all other fields than the login fields auth is required. So please remove it also on all others.